ts/DataGrid/Options.ts

```diff
@@ -391,6 +397,14 @@ export interface HeaderEvents {
     click?: ColumnEventCallback;
 }
 
+export interface GroupedHeader {
```

Comment:
Describe everything. Do we need the `level`?

Reply:
Yes, for saving level when we iterate over the whole header's tree.